packages/aws-rfdk/lib/deadline/test/usage-based-licensing.test.ts

```diff
@@ -103,29 +110,71 @@ describe('UsageBasedLicensing', () => {
     licenses = [UsageBasedLicense.forMaya()];
   });
 
-  test('creates an ECS cluster', () => {
-    // WHEN
-    new UsageBasedLicensing(stack, 'UBL', {
+  function createUbl(props?: Partial<UsageBasedLicensingProps>): UsageBasedLicensing {
+    return new UsageBasedLicensing(stack, DEFAULT_CONSTRUCT_ID, {
       certificateSecret,
       images,
       licenses,
       renderQueue,
       vpc,
+      ...props,
     });
+  }
+
+  describe('configures auto registration', () => {
+    test('default to private subnets', () => {
+      // WHEN
+      const ubl = createUbl();
+
+      // THEN
+      const expectedCall: SubnetIdentityRegistrationSettingsProps = {
+        dependent: ubl.service.node.defaultChild as CfnService,
+        registrationStatus: SecretsManagementRegistrationStatus.REGISTERED,
+        role: SecretsManagementRole.CLIENT,
+        vpc,
+        vpcSubnets: { subnetType: SubnetType.PRIVATE },
+      };
+
+      // THEN
+      expect(renderQueue.configureSecretsManagementAutoRegistration).toHaveBeenCalledWith(expectedCall);
+    });
+
+    test.each<[SubnetSelection]>([
+      [{
+        subnetType: SubnetType.PUBLIC,
```

Comment:
Do we need to use `test.each` here? Can we simplify this to just be a single test?